backend/controllers/LocationsController.ts

```diff
@@ -94,34 +94,42 @@ const LocationsController = {
       return res.status(500).json({ error: err.message });
     }
   },
-  async changeLocationInfo(req: Request, res: Response) {
+
+  async changeLocationData(req: Request, res: Response) {
     try {
-      const { _id, fields } = req.body;
+      let { id } = req.params;
+      let { locationName, description } = req.body;
 
-      const location = await Location.findById(_id);
```

Comment:
will there be an opportunity to change location filters??
if yes - they should be passed in the body as well